### PR TITLE
Possible PPS mode implementation

### PR DESCRIPTION
**Pull request by zhq1**
Possible PPS mode implementation

### CODE REVIEW

**Comment by goodthebest** [comment]
Did you test this implementation of PPS, whether payouts were accordingly?

**Comment by phm87** [comment]
In the new code of BackendSharesNew for PPS, I don't see YAAMP_ALLOW_EXCHANGE.

In the existing code of BackendSharesNew, there is a difference between with autoexchange and without autoexchange on earning accounting and share deletion : if only 1 coin is mined, calculations are performed per coin (profit per coin) but if autoexchange is enabled, calculations are performed per algo (profit per algo).

In the new PPS code you propose, did you foreseen code for autoexchange ? Did you test with/without autoexchange ?

File: web/yaamp/core/backend/blocks.php [hunk 1]
```diff
@@ -78,9 +78,59 @@ function BackendBlockNew($coin, $db_block)
 		// [*:message] => 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1205 Lock wait timeout exceeded; try restarting transaction'
 	}
 }
-
+	
+｝
```
Comment (Heraisu) [inline web/yaamp/core/backend/blocks.php:82]: This character is not parsed in php7.2. Was supposed to be a } ?

**Comment by rlinxy** [comment]
> Possible PPS mode implementation

你好，我们在运营矿工社区和矿池，能否加个微信或者QQ，看看是否有合作的可能，账号都是40984222，谢谢

**Pull request closed without merging**